src/PanoViewer/PanoViewer.js

```diff
@@ -57,8 +58,19 @@ export default class PanoViewer extends Component {
 			return this;
 		}
 
+		if (!!options.image && !!options.video) {
```

Comment:
@happyhj I had changed above condition `if (!!options.image === !!options.video)`. but I changed it . because of allowing no content specified.